frontend/src/components/abfragevarianten/AbfragevarianteFormular.vue

```diff
@@ -380,5 +409,14 @@ export default class AbfragevarianteForm extends Mixins(
       this.abfragevariante.geschossflaecheSonstiges = undefined;
     }
   }
+
+  private datumSatzungsbeschlussChanged(datumSatzungsbeschluss: Date): void {
+    if (!_.isNil(datumSatzungsbeschluss)) {
+      this.abfragevariante.realisierungVon =
```

Comment:
In der Userstory ist nicht definiert, wie sich das Feld Realisierung von in folgenen Szenario verhalten soll.

1. `datumSatzungsbeschluss` und `realisierungVon ` sind leer.
2. `realisierungVon ` wird manuell  befüllt und `datumSatzungsbeschluss` bleibt leer.
3.  Nun wird `datumSatzungsbeschluss` befüllt.

Wie soll sich Realisierung von verhalten? Wird der Wert immer überschrieben (egal ob realisierungVon leer oder befüllt )? Oder findet keine Überschreibung des Attributs realisierungVon  statt, wenn bereits ein Wert im Feld realisierungVon gepflegt ist?

Reply:
Nach Rücksprache mit Valerie kann bei dem beschriebenen Scenario _Realisierung von_ nach  der Eingabe  von _Datum Satzungsbeschluss_ ohne Rückmeldung überschrieben werden. Darauf weist der Hilfe-Text hin.